lib/notification-worker.js

```diff
@@ -60,6 +60,21 @@ process.on('message', (m) => {
             }
             process.send({change: m.change});
             break;
+        case 'startup':
+            if (impl['on' + m.message]) {
```

Comment:
The `'on' + m.message` construction is used above because there are multiple messages being handled. That is not the case here.